armbotoffboard/subsystems/drivesubsystem.py

```diff
@@ -51,23 +52,25 @@ def __init__(self) -> None:
         # gearbox is constructed, you might have to invert the left side instead.
         self.right.setInverted(True)
 
-    def arcadeDrive(
+    def arcadeDriveCommand(
         self, fwd: float, rot: float
-    ) -> None:
-        """Drives the robot using arcade controls.
+    ) -> commands2.Command:
+        """
+        A split-stick arcade command, with forward/backward controlled by the left hand, and turning
+        controlled by the right.
 
         Args:
-            fwd: The commanded forward movement
-            rot: The commanded rotation
+            fwd: supplier for the commanded forward movement
+            rot: supplier for the commanded rotation
```

Comment:
The type hints for `fwd` and `rot` indicate these aren't suppliers.

Reply:
In [Java](https://github.com/wpilibsuite/allwpilib/blob/main/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/armbotoffboard/subsystems/DriveSubsystem.java), is indicated that these parameters are suppliers.
Should I change the comments to how it was before?

Reply:
Please make this behave the same as the Java and [C++](https://github.com/wpilibsuite/allwpilib/blob/2744991771c0e2d0d9d06a12e2b5dd7dcaf9e78e/wpilibcExamples/src/main/cpp/examples/ArmBotOffboard/include/subsystems/DriveSubsystem.h) example.

Reply:
let me see if I'm doing right: Is C++'s `std::function<double()>` the equivalent to Python's `typing.Callable[[], float]` ?